packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -451,6 +461,23 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          {type !== 's3' && (
+            <fieldset className='form-group form-group'>
+              <div className='input-group form-group'>
+                <span className='align-middle'>
+                  {_('remoteUseVhdDirectory')}
+                  <Tooltip content={_('remoteUseVhdDirectoryTooltip')}>
+                    <Icon icon='info' size='lg' />
+                  </Tooltip>
+                </span>
+                <Toggle
+                  className='align-middle pull-right'
+                  onChange={effects.setInsecure}
+                  value={useVhdDirectory === true}
```

Comment:
Did you mean `effects.setUseVhdDirectory` ?
It weird to have for value `useVhdDirectory === true` and the `onChange` function that update the `protocol` and not the `useVhdDirectory`.

It is intended ?

Reply:
this part of the form is not visible if you are on a  s3 filesystem